lib/views/pages/profile/posts_info/info_card_post.dart

```diff
@@ -0,0 +1,48 @@
+import "package:flutter/material.dart";
+
+//info card for the badges
+class InfoCardPost extends StatelessWidget {
+  static const cardKey = Key("card");
+
+  const InfoCardPost({
+    super.key,
+    required this.shadow,
+  });
+
+  final BoxShadow shadow;
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      key: cardKey,
+      width: 54,
+      height: 80,
+      decoration: BoxDecoration(
+        color: Theme.of(context).colorScheme.secondaryContainer,
+        borderRadius: const BorderRadius.all(Radius.circular(10)),
+        boxShadow: [shadow],
+      ),
+      child: Center(
+        child: ListTile(
+          title: //TODO: replace with real data
+              Text(
+            "Post Title",
+            style: Theme.of(context).textTheme.titleSmall,
+          ),
+          subtitle: Text(
+            "My super post that talks about something that is super cool and is located in a super spot",
+            style: Theme.of(context).textTheme.bodySmall,
+            maxLines: 2,
+            overflow: TextOverflow.ellipsis,
+          ),
+          trailing: const IconButton(
+            icon: Icon(
+              Icons.delete,
+              size: 32,
+            ),
+            onPressed: null,
```

Comment:
I think we could add just a little comment here.
```suggestion
            ),
            // TODO : add the logic for deleting a post
            onPressed: null,
```